src/pipecat/processors/frameworks/rtvi.py

```diff
@@ -1244,6 +1579,39 @@ async def _handle_update_config(self, request_id: str, data: RTVIUpdateConfig):
         await self._update_config(RTVIConfig(config=data.config), data.interrupt)
         await self._handle_get_config(request_id)
 
+    async def _handle_update_context(self, data: RTVIAppendToContextData, request_id: str):
+        if data.role not in ["user", "assistant"]:
+            logger.warning(f"Unknown role {data.role} in RTVIAppendToContext")
+            await self._send_error_response(
+                request_id, f"Invalid role {data.role} for RTVIAppendToContext"
+            )
```

Comment:
should we return here?

Reply:
Actually, maybe we should remove this check. In theory we have this protection:

```
    role: Literal["user", "assistant"] | str
```

And `str` could be used to set any role. I think it's more generic if we remove this block.